plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -175,10 +176,12 @@ def _convert_replica_spec(
         if not isinstance(replica_config, Master):
             replicas = replica_config.replicas
         return pytorch_task.DistributedPyTorchTrainingReplicaSpec(
-            replicas=replicas,
-            image=replica_config.image,
-            resources=resources.to_flyte_idl() if resources else None,
-            restart_policy=replica_config.restart_policy.value if replica_config.restart_policy else None,
+            common=plugins_common.CommonReplicaSpec(
+                replicas=replicas,
```

Comment:
It's not backward compatible. Here, we still need to specify replicas, image, etc.